src/libs/API.js

```diff
@@ -27,13 +26,12 @@ let isAuthenticating;
 function handleExpiredAuthToken(originalCommand, originalParameters, originalType) {
     // When the authentication process is running, and more API requests will be requeued and they will
     // be performed after authentication is done.
-    if (isAuthenticating) {
+    if (NetworkStore.getIsAuthenticating()) {
         return Network.post(originalCommand, originalParameters, originalType);
     }
 
     // Prevent any more requests from being processed while authentication happens
-    Network.pauseRequestQueue();
-    isAuthenticating = true;
+    NetworkStore.setIsAuthenticating(true);
```

Comment:
NAB: I think we can use `NetworkStore.isAuthenticating()` and `NetworkStore.setAuthenticating(value)` for this boolean value, no? https://airbnb.io/javascript/#accessors--boolean-prefix

Reply:
Maybe - though unsure if we follow that guidance strictly. I had it like that but changed it as I couldn't think of a way to do that without changing the variable name here...

https://github.com/Expensify/App/blob/c39f2cfdbb9886a10942aec6cc3c05ec4375b170/src/libs/Network/NetworkStore.js#L11

Reply:
I wouldn't see a problem renaming it to `authenticating` since it's only used as a private variable and the `NetworkStore` is a small file, but that's just my opinion :)